packages/langium-cli/src/generate.ts

```diff
@@ -6,8 +6,7 @@
 
 import fs from 'fs-extra';
 import {
-    AbstractRule, createLangiumGrammarServices, getDocument, Grammar, Interface, isAbstractRule, isGrammar, isInterface, isParserRule, isType,
-    LangiumDocument, linkContentToContainer, Type
+    createLangiumGrammarServices, getDocument, Grammar, GrammarAST, LangiumDocument, linkContentToContainer
 } from 'langium';
```

Comment:
I like the prefixing idea 'GrammarAST.<...>'. But the naming is not very 1:1. I mean `LangiumAST` would be more direct, whereas `GrammarAST` does not tell me what grammar is meant, or?

Grammar -> LangiumGrammar
GrammarAST -> LangiumAST
LangiumDocument -> perfect

Reply:
Right, but then I'd name it `LangiumGrammarAST`. LangiumAST is still ambiguous.

Reply:
Isn't `LangiumGrammarAST` too long?

Reply:
@pluralia you are right, too. So then I'd leave the namespace as `GrammarAST` for now. We should rethink our exported API on a more general level. Ideas so far:

 - Create focused index files in every subfolder, narrow the main index (https://github.com/langium/langium/discussions/155)
 - Create a hierarchy of namespaces like the [vscode API](https://code.visualstudio.com/api/references/vscode-api)
 - Keep the current global index and ensure proper naming of exported things

Reply:
Should we put these ideas in a separate issue?

Reply:
I don't know whether it's really an issue. I'd make a discussion and turn it into an issue if we find problems with the current approach.